core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2982,6 +2995,10 @@ struct ViewValueFunctor<DeviceType, ValueType, true /* is_scalar */> {
                    size_t const arg_n, std::string arg_name)
       : space(arg_space), ptr(arg_ptr), n(arg_n), name(std::move(arg_name)) {}
 
+  ViewValueFunctor(ValueType* const arg_ptr, size_t const arg_n,
+                   std::string arg_name)
+      : space(ExecSpace{}), ptr(arg_ptr), n(arg_n), name(std::move(arg_name)) {}
```

Comment:
It looks like initialization for `default_exec_space` is missing here, in the old and new overloads?

Reply:
See dfa6b1b5ba4818ea5f4fc8436573914675371f98.